types/auth0/index.d.ts

```diff
@@ -1295,6 +1299,10 @@ export class OrganizationsManager {
     getAll(cb: (err: Error, organizations: Organization[]) => void): void;
     getAll(params: PagingOptions): Promise<Organization[]>;
     getAll(params: PagingOptions, cb: (err: Error, organizations: Organization[]) => void): void;
```

Comment:
Since you're already covering these in your overload with the optional `include_totals?: false` would it make sense to remove these?
```suggestion
```

Reply:
Ah good call, removed those @nhardy.